test/unit/org/apache/cassandra/index/sasi/SASICQLTest.java

```diff
@@ -348,4 +351,21 @@ public void testIntCompare() throws Throwable
             }
         }
     }
+
+    @Test
+    public void testInOperator() throws Throwable
+    {
+        createTable("CREATE TABLE %s (pk int primary key, v int);");
+
+        createIndex(String.format("CREATE CUSTOM INDEX ON %%s (v) USING 'org.apache.cassandra.index.sasi.SASIIndex' WITH OPTIONS = {'mode': 'CONTAINS'};"));
+
+        execute("INSERT INTO %s (pk, v) VALUES (?, ?);", 0, 100);
+        execute("INSERT INTO %s (pk, v) VALUES (?, ?);", 1, 200);
+        execute("INSERT INTO %s (pk, v) VALUES (?, ?);", 2, 300);
+
+        assertInvalidThrowMessage(Optional.of(ProtocolVersion.CURRENT),
+                                  StatementRestrictions.REQUIRES_ALLOW_FILTERING_MESSAGE,
+                                  InvalidQueryException.class, 
+                                  "SELECT * FROM " + KEYSPACE + '.' + currentTable() + " WHERE v IN (200, 250, 300)");
```

Comment:
We can use a `%s` placeholder for the query
```suggestion
                                  StatementRestrictions.REQUIRES_ALLOW_FILTERING_MESSAGE,
                                  InvalidQueryException.class, 
                                  "SELECT * FROM %s WHERE v IN (200, 250, 300)");
```